modules/atlas/src/main/resources/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0.json

```diff
@@ -0,0 +1,51 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/atlas/parcellationmeshgeneration/v0.1.0/shapes/"
+    }
+  ],
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0"
+  ],
+  "@type": "nxv:Schema",
+  "shapes": [
+    {
+      "@id": "this:ParcellationMeshGenerationShape",
+      "@type": "sh:NodeShape",
+      "label": "Parcellation mesh generation activity description.",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "targetClass": "nsg:ParcellationMeshGeneration",
+      "and": [
+        {
+          "@id": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.0/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "prov:used",
+              "description": "Parcellation volume",
+              "class": "nsg:ParcellationVolume",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "prov:generated",
+              "class": "nsg:BrainParcellationMesh",
+              "minCount": 1,
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:brainRegion",
```

Comment:
I suggest to align and normalize the way brain region is stored. What about having the brainRegion in the brain parcellation mesh entity embedded in a brainLocation object ?